### PR TITLE
[player-1264] Changing VAST and IMA to leave calculating ad positions…

### PR DESCRIPTION
**Pull request by pilievOoyala**
…, that are percentages, up to the AMC.  They will just pass along the position type and the duration of the content.

This needs to be merged at the same time as https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/pull-requests/307/overview

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -416,13 +416,15 @@ require("../html5-common/js/utils/utils.js");
                   "adManager": this.name,
                   "ad": ad,
                   "streams": streams,
-                  "adType": _amc.ADTYPE.UNKNOWN_AD_REQUEST
+                  "adType": _amc.ADTYPE.UNKNOWN_AD_REQUEST,
+                  "mainContentDuration": this.mainContentDuration
                 };
 
                 //percentage position types require a different calculation.
```
Comment (jpfranco) [inline js/google_ima.js]: Would probably just update this comment to say that this is handled by the AMC, looks good otherwise.

Comment (pilievOoyala) [reply js/google_ima.js]: Removed. since it no longer applies.

**Comment by jpfranco** [comment]
LGTM

File: js/ad_manager_vast.js [hunk 2]
```diff
@@ -909,21 +909,25 @@ OO.Ads.manager(function(_, $) {
...
               adData.position = +adMetadata.time / 1000;
             } else if (_isValidPosition(adMetadata.position)) {
               adData.position = +adMetadata.position / 1000;
             }
-          } else if (adMetadata.position_type == 'p') {
+          } else if (adMetadata.position_type === 'p') {
             if (_isValidPosition(adMetadata.position)) {
-              adData.position = +adMetadata.position / 100 * this.mainContentDuration;
+              adData.positionType = adMetadata.position_type;
+              adData.position = +adMetadata.position;
```
Comment (aldigun) [inline js/ad_manager_vast.js]: what is = + operation?

**Pull request merged**